### DISCUSSION TITLE
Price analyzer with Norwegian subsidy

### DISCUSSION THREAD

**Discussion by rune454** [erlendsellie/priceanalyzer#35] [Q&A]
I've been trying to implement the Norwegian subsidy using the 'Example with Tensio 2023 Grid prices' under 'Additional cost'. Is there anything else required to be done beside coping the text in 'Template for additional costs'? I can't seem to make this work.

Thanks.

**Comment by erlendsellie** [comment]
Hi!
The example provided is working in the beta version of this integration, which is a bit delayed as as full release. You should be able to upgrade thorugh HACS->redowload->show beta versions. 

You may have to configure Priceanalyzer again. Just click configure on the integration and save, if it's not working after upgrading.

Then try the template for subsidy.
